DigitalLearningSolutions.Web/Views/Shared/_UserFeedbackBarPartial.cshtml

```diff
@@ -15,7 +16,7 @@
                     </div>
                     <a asp-controller="UserFeedback" asp-action="Index" asp-route-sourceurl="@Context.Request.GetDisplayUrl()"
                asp-route-sourcePageTitle="@(ViewData.ContainsKey("Title") ? ViewData["Title"].ToString() : string.Empty)">Give page feedback</a>
-                    or <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=K5Gn_5ewMUGcD9DoB1Wyqy_N2dia4StEr8rvzDM1MnlUNEhQUDk4UVpHUDc2N0g4OTZGNFRBNktLOC4u" target="_blank">join our research panel</a> - this website is in
+                    or <a href="@Model" target="_blank">join our research panel</a> - this website is in
```

Comment:
It should be @Model.UserResearchUrl

Reply:
This can be @Model.UserResearchUrl it is @model cause am passing a string there not object